src/resources/locales/en.json

```diff
@@ -188,36 +188,36 @@
             "urlPlaceholder": "Link"
         },
         "addFormApiapp": {
-            "title": "Add a library account"
+            "title": "Add library account"
         },
         "addMenu": "Add OPDS feed",
         "breadcrumbRoot": "Catalogs",
         "documentation": "OPDS official website",
         "empty": "No publication found",
         "firstPage": "First",
-        "informations": "The Open Publication Distribution System (OPDS) catalogue format is a way to distribute, discover, and download electronic publications.",
+        "informations": "Imagine a world where you can explore and download digital publications from public libraries and digital booksellers with just a few clicks. That's exactly what the Open Publication Distribution System (OPDS) catalog format offers. With OPDS, you can navigate through catalogs from various ebook providers, sign up, borrow, buy a publication, or visit your personal bookshelf. You can even download an ebook or audiobook directly into Thorium Reader for instant reading.\nSo, how do you find an OPDS catalog? It's simple. Look for a hyperlink (URL) leading to the catalog. Copy this URL into the Link field above. Keep in mind that there's no one-stop shop for all active OPDS catalogs. You'll need to contact your preferred library or bookseller and ask for their OPDS catalog URL. If you're not sure where to start, Wikipedia has a few registries of OPDS catalogs in various languages that might be helpful.\nThe Name field is left at your convenience; it provides an easy way to identify the feed in Thorium Reader.",
         "lastPage": "Last",
         "menu": {
-            "aboutBook": "About this book",
-            "addExtract": "Import the extract",
-            "goBuyBook": "Buy this book",
-            "goLoanBook": "Borrow this book",
+            "aboutBook": "About publication",
+            "addExtract": "Import extract",
+            "goBuyBook": "Buy publication",
+            "goLoanBook": "Borrow publication",
             "goRevokeLoanBook": "Revoke loan",
```

Comment:
I'm surprised to find this label: IMO it should not exist

Reply:
It's an OPDS thing, not LCP...let me check

Reply:
`http://librarysimplified.org/terms/rel/revoke`